lib/ts/recipe/passwordless/smsDelivery.ts

```diff
@@ -0,0 +1,27 @@
+// /* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
You can remove this file, as it won't be needed when implementing SMS delivery anyway

Reply:
okay

Reply:
Not done